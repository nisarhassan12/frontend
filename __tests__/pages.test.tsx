import React from 'react'
import { create } from 'react-test-renderer'
import User from '../pages/user'
import Register from '../pages/register'

it('Random user page renders correctly', () => {
  const tree = create(<User id="Ivan" />).toJSON()
  expect(tree).toMatchSnapshot()
})

it('Register page renders correctly', () => {
  const tree = create(<Register />).toJSON()
  expect(tree).toMatchSnapshot()
})
