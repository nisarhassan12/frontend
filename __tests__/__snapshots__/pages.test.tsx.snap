// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Random user page renders correctly 1`] = `
<div
  className="pages__Page-sc-9mrxf5-0 bOFcJn"
>
  <p>
    Hello, #
    Ivan
    !
  </p>
  <p>
    There will be content soon.
  </p>
  <p>
    You need to wait a bit until we release our pre-alpha.
  </p>
  <a
    href="/"
    onClick={[Function]}
    onMouseEnter={[Function]}
  >
    Home
  </a>
</div>
`;

exports[`Register page renders correctly 1`] = `
<div
  className="sc-bdVaJa sc-bwzfXH register___StyledFlex-sc-1cljw8j-1 iJKweu"
>
  <div
    className="sc-bdVaJa sc-bwzfXH NavBar__StyledNav-sc-5bxc2y-1 jGuGMY"
    width={1}
  >
    <a
      href="/"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
      <img
        className="sc-bdVaJa sc-bZQynM ilartJ"
        height="7vh"
        src="../../static/logo.svg"
        width="100%"
      />
    </a>
    <div
      className="NavBar__LinksGroup-sc-5bxc2y-0 blbCBZ"
    >
      <a
        href="/login"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        Sign In
      </a>
      <a
        href="/register"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        Sign Up
      </a>
    </div>
  </div>
  <h1
    className="sc-bdVaJa sc-htpNat sc-bxivhb register__Heading-sc-1cljw8j-0 cZiABb"
    fontSize={4}
    fontWeight="bold"
  >
    Hello 
    
    !
  </h1>
  <div
    className="sc-bdVaJa sc-gzVnrw jbeNhY"
  >
    <form
      action="https://api.komfy.now.sh/reg"
      className="Form__StyledForm-sc-1qkwnfg-0 bGzgGo"
      method="POST"
      onSubmit={[Function]}
    >
      <input
        name="username"
        onInput={[Function]}
        placeholder="How we can call you?"
        type="text"
      />
      <input
        name="email"
        placeholder="hello@example.com"
        type="email"
      />
      <button
        onClick={[Function]}
      >
        Create password
      </button>
      <div>
        Complex password:
        <input
          onChange={[Function]}
          type="checkbox"
          value="off"
        />
      </div>
      
      <input
        disabled={true}
        name="password"
        value=""
      />
    </form>
  </div>
</div>
`;
